dotnet/packages/Microsoft.Bot.Builder.M365/Microsoft.Bot.Builder.M365.Tests/.editorconfig

```diff
@@ -3,5 +3,4 @@
 dotnet_diagnostic.CA2000.severity = none # Call System.IDisposable.Dispose on object before all references to it are out of scope
 dotnet_diagnostic.CA2007.severity = none # Do not directly await a Task
 dotnet_diagnostic.IDE0008.severity = none # Use explicit type instead of var
-dotnet_diagnostic.IDE0058.severity = none # Remove unnecessary expression value
 dotnet_diagnostic.VSTHRD111.severity = none # Use .ConfigureAwait(bool)
```

Comment:
Are these like linting rules? Why are we removing this line?

Reply:
Yep, added it back since it helps detect unnecessary calls.
BTW, still disable it for test project since in test there will be a lot `Assert...` calls.